### PR TITLE
Update multiple components

### PR DESCRIPTION
**Pull request by Brentdevent**

### CODE REVIEW

File: src/client/component/bots.cpp [hunk 1]
```diff
@@ -118,7 +118,7 @@ namespace bots
 		void post_unpack() override
 		{
 			utils::hook::jump(game::select(0x141653B70, 0x1402732E0), get_bot_name);
```
Comment (momo5502) [inline src/client/component/bots.cpp:120]: what about this?

File: src/client/component/network.cpp [hunk 2]
```diff
@@ -237,22 +237,22 @@ namespace network
...
-			utils::hook::nop(game::select(0x142332E76, 0x140596DF6), 4); // don't increment data pointer to optionally skip socket byte
-			utils::hook::call(game::select(0x142332E43, 0x140596DC3), read_socket_byte_stub); // optionally read socket byte
-			utils::hook::call(game::select(0x142332E81, 0x140596E01), verify_checksum_stub); // skip checksum verification
-			utils::hook::set<uint8_t>(game::select(0x14233305E, 0x140596F2E), 0); // don't add checksum to packet
+			utils::hook::nop(game::select(0x1423322B6, 0x140596DF6), 4); // don't increment data pointer to optionally skip socket byte
+			utils::hook::call(game::select(0x142332283, 0x140596DC3), read_socket_byte_stub); // optionally read socket byte
+			utils::hook::call(game::select(0x1423322C1, 0x140596E01), verify_checksum_stub); // skip checksum verification
+			utils::hook::set<uint8_t>(game::select(0x14233249E, 0x140596F2E), 0); // don't add checksum to packet
 
 			utils::hook::set<uint32_t>(game::select(0x14134C6E0, 0x14018E574), 5); // set initial connection state to challenging
```
Comment (momo5502) [inline src/client/component/network.cpp:245]: this?

File: src/client/component/scheduler.cpp [hunk 3]
```diff
@@ -168,12 +168,12 @@ namespace scheduler
 		{
 			if (!game::is_server())
 			{
-				r_end_frame_hook.create(0x142273560_g, r_end_frame_stub);
+				r_end_frame_hook.create(0x142272B00_g, r_end_frame_stub);
 				// some func called before R_EndFrame, maybe SND_EndFrame?
 			}
 
 			//g_run_frame_hook.create(0x14065C360_g, server_frame_stub); // GlassSv_Update
```
Comment (momo5502) [inline src/client/component/scheduler.cpp:175]: can you remove the comment

**Review by momo5502 — CHANGES_REQUESTED** [review]

File: src/client/component/bots.cpp (hunk 1 above)
Comment (Brentdevent) [reply src/client/component/bots.cpp:120]: address hasn't changed

File: src/client/component/network.cpp (hunk 2 above)
Comment (Brentdevent) [reply src/client/component/network.cpp:245]: address hasn't changed

File: src/client/component/bots.cpp (hunk 1 above)
Comment (momo5502) [reply src/client/component/bots.cpp:120]: for real xD

**Review by momo5502 — APPROVED** [review]

File: src/client/component/bots.cpp (hunk 1 above)
Comment (Brentdevent) [reply src/client/component/bots.cpp:120]: Yep haha, a lot of stuff in ui_scripting also had the same address

**Pull request merged**